schemas/representations.md

```diff
@@ -149,102 +244,241 @@ type Fizzlebop struct {
 }
 ```
 
-Some data matching the `Fizzlebop` struct (shown as JSON) is:
+Some data matching the `Fizzlebop` Struct (shown as JSON) is:
 
 ```json
 "value-of-a:value-of-b"
 ```
 
-Since this is a struct, and none of the fields are optional, it *must* have two fields:
-therefore, a string with no `":"` characters would be rejected as not matching.
+Since there is no escaping mechanism, no value can contain the string used for `join` as this would disrupt decoding and result in an error. Additionally, optional or implicit fields are not possible with the `stringjoin` Struct representation strategy, all elements must be present.
+
+No field-specific parameters are available for the `stringjoin` Struct representation strategy (i.e. `implicit` and `rename` which are available for `map`). The single general parameter is mandatory:
+
+* `join` declares a quoted string that is used to delimit field values
+
+### Struct `listpairs` Representation
 
-### map stringpairs representation
+**Representation Kind: List**
 
-Say we're doing something awfully like the mount options in an /etc/fstab file:
+The `listpairs` Struct representation strategy encodes a Struct to a List of Lists at the Data Model layer. Similar to the `tuple` representation but the keys are also included in the encoding so the serialized form is not as opaque and more self-describing than `tuple`.
+
+**Example**
+
+```ipldsch
+type Foo struct {
+	fieldOne String
+	fieldTwo Bool
+} representation listpairs
+```
+
+Some data matching the `Foo` Struct (shown as JSON) is:
+
+```json
+[["fieldOne", "this is field one"], ["fieldTwo", true]]
+```
+
+Each entry of the parent List is a List containing exactly two entries, where the first is the exact field name and the second is the value.
+
+No field-specific parameters are available for the `listpairs` Struct representation strategy (i.e. `implicit` and `rename` which are available for `map`), although these may be introduced in the future since they are technically possible with this strategy. No general parameters are available.
```

Comment:
It is true of `stringpairs`, `stringjoin` (`implicit` only) as well. So perhaps this paragraph could be put in the **Representation Kind: Map** section that field-specific parameters are only to the map-representation and not for others, but it might be added in the future.

Reply:
yeah, it gets iffy when you move outside of the key=value representations, but still possible. Will move it like you suggest though.